src/main/java/com/uber/cadence/internal/sync/TestActivityEnvironmentInternal.java

```diff
@@ -18,71 +18,7 @@
-import com.uber.cadence.DescribeWorkflowExecutionResponse;
-import com.uber.cadence.DomainAlreadyExistsError;
-import com.uber.cadence.DomainNotActiveError;
-import com.uber.cadence.EntityNotExistsError;
-import com.uber.cadence.GetSearchAttributesResponse;
-import com.uber.cadence.GetWorkflowExecutionHistoryRequest;
-import com.uber.cadence.GetWorkflowExecutionHistoryResponse;
-import com.uber.cadence.InternalServiceError;
-import com.uber.cadence.LimitExceededError;
-import com.uber.cadence.ListClosedWorkflowExecutionsRequest;
-import com.uber.cadence.ListClosedWorkflowExecutionsResponse;
-import com.uber.cadence.ListDomainsRequest;
-import com.uber.cadence.ListDomainsResponse;
-import com.uber.cadence.ListOpenWorkflowExecutionsRequest;
-import com.uber.cadence.ListOpenWorkflowExecutionsResponse;
-import com.uber.cadence.ListWorkflowExecutionsRequest;
-import com.uber.cadence.ListWorkflowExecutionsResponse;
-import com.uber.cadence.PollForActivityTaskRequest;
-import com.uber.cadence.PollForActivityTaskResponse;
-import com.uber.cadence.PollForDecisionTaskRequest;
-import com.uber.cadence.PollForDecisionTaskResponse;
-import com.uber.cadence.QueryFailedError;
-import com.uber.cadence.QueryWorkflowRequest;
-import com.uber.cadence.QueryWorkflowResponse;
-import com.uber.cadence.RecordActivityTaskHeartbeatByIDRequest;
-import com.uber.cadence.RecordActivityTaskHeartbeatRequest;
-import com.uber.cadence.RecordActivityTaskHeartbeatResponse;
-import com.uber.cadence.RegisterDomainRequest;
-import com.uber.cadence.RequestCancelWorkflowExecutionRequest;
-import com.uber.cadence.ResetStickyTaskListRequest;
-import com.uber.cadence.ResetStickyTaskListResponse;
-import com.uber.cadence.ResetWorkflowExecutionRequest;
-import com.uber.cadence.ResetWorkflowExecutionResponse;
-import com.uber.cadence.RespondActivityTaskCanceledByIDRequest;
-import com.uber.cadence.RespondActivityTaskCanceledRequest;
-import com.uber.cadence.RespondActivityTaskCompletedByIDRequest;
-import com.uber.cadence.RespondActivityTaskCompletedRequest;
-import com.uber.cadence.RespondActivityTaskFailedByIDRequest;
-import com.uber.cadence.RespondActivityTaskFailedRequest;
-import com.uber.cadence.RespondDecisionTaskCompletedRequest;
-import com.uber.cadence.RespondDecisionTaskCompletedResponse;
-import com.uber.cadence.RespondDecisionTaskFailedRequest;
-import com.uber.cadence.RespondQueryTaskCompletedRequest;
-import com.uber.cadence.ServiceBusyError;
-import com.uber.cadence.SignalWithStartWorkflowExecutionRequest;
-import com.uber.cadence.SignalWorkflowExecutionRequest;
-import com.uber.cadence.StartWorkflowExecutionRequest;
-import com.uber.cadence.StartWorkflowExecutionResponse;
-import com.uber.cadence.TerminateWorkflowExecutionRequest;
-import com.uber.cadence.UpdateDomainRequest;
-import com.uber.cadence.UpdateDomainResponse;
-import com.uber.cadence.WorkflowExecution;
-import com.uber.cadence.WorkflowExecutionAlreadyStartedError;
+import com.uber.cadence.*;
```

Comment:
Please consider removing the start import and import all classes independently

Reply:
See also the following classes: 
- src/main/java/com/uber/cadence/internal/sync/TestWorkflowEnvironmentInternal.java
- src/main/java/com/uber/cadence/internal/testservice/TestWorkflowService.java